test/correctness/simd_op_check.cpp

```diff
@@ -1746,13 +1746,8 @@ class SimdOpCheck : public SimdOpCheckTest {
                     check("v128.const", 8 * w, f32_1 * f32(42));
                     check("v128.const", 4 * w, f64_1 * f64(42));
                 } else {
-                    if (Halide::Internal::get_llvm_version() == 120) {
```

Comment:
This also looks like an llvm 12 block

Reply:
it is, but we are keeping it

Reply:
But it's an == 120 block, not a >= 120 block, so by keeping it aren't we testing the wrong thing on llvm 13?

Or is the original check wrong?

Reply:
bump. I'm confused about why this isn't going to cause simd op check failures on llvm 13+

Or were we only testing wasm on llvm 12 before?

Reply:
Old code:
```
                    if (Halide::Internal::get_llvm_version() == 120) {
                        check("i64x2.splat", 8 * w, u16_1 * u16(42));
                        check("i64x2.splat", 4 * w, u32_1 * u32(42));
                    } else {
                        check("i16x8.splat", 8 * w, u16_1 * u16(42));
                        check("i32x4.splat", 4 * w, u32_1 * u32(42));
                   }
```

New code:

```
                    check("i64x2.splat", 8 * w, u16_1 * u16(42));
                    check("i64x2.splat", 4 * w, u32_1 * u32(42));
```

I think the new code is just the llvm_version() == 120 version, so aren't we changing the expected output for llvm 13/14?

I can't figure out what I'm missing.